03week/ticTacToe.js

```diff
@@ -24,23 +24,53 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  // Possible horizontal wins
+  let winArrH = board[0][0];
+  if ((board[0][0] == playerTurn && board[0][1] == playerTurn && board[0][2] == playerTurn) || (board[1][0] == playerTurn && board[1][1] == playerTurn && board[1][2] == playerTurn) || (board[2][0] == playerTurn && board[2][1] == playerTurn && board[2][2] == playerTurn)) {
```

Comment:
@AFutureWebDev 
When lines get this long, you'll want to break it up into multiple lines to make it more readable